src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1167,4 +1167,7 @@ Win + Shift + O to toggle your video</value>
   <data name="Run_Radio_Position_Primary_Monitor.Content" xml:space="preserve">
     <value>Primary monitor</value>
   </data>
-</root>
+  <data name="Run_PluginsLoading.Text" xml:space="preserve">
+    <value>Plugins are loading...</value>

```

Comment:
```suggestion
    <value>Plugins are loading.</value>
```
and adding a circle with spinning dots befor the text.

Reply:
Are you suggesting adding ProgressRing?

Reply:
Or we can use progres dots like this:
`Plugins are loading ••• •• •`

(I hope you know what I mean.)

Reply:
`Plugins are loading...` is good enough, also consider that this is an edge case and 99% of the users will never see it.

Reply:
<value>Plugins are loading...</value>

Reply:
Let's leave it as it is. Take into account that ProgressRing is not easy to integrate into the message so it looks good.